tests/acceptance/features/webUIFiles/search.feature

```diff
@@ -153,3 +153,20 @@ Feature: Search
     And the user searches for "favorite" using the webUI
     Then folder "favorite folder" should be listed on the webUI
     And folder "not favorite folder" should be listed on the webUI
+
+  @issue-3044
+  Scenario: Search for file that has same name as deleted file in trashbin
```

Comment:
is this about searching a deleted file or delete a searched file, or both?
If both, maybe we need to split it

Reply:
Its about deleting a search file which has name similar to a file that is in trashbin.